sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/WhereQuery.cs

```diff
@@ -117,16 +201,24 @@ public override AdtQueryBuilder Build()
         /// <inheritdoc/>
         public override string Stringify()
         {
-            // TODO -- make this cleaner? design problem?
             if (_clauses.Any())
             {
-                // TODO -- if appending more strings turn into a stringbuilder
-                string whereComponents = $"{QueryConstants.Where} ";
-
-                // TODO -- turn condition into a string
-                // add where arguments (conditions)
-
-                return whereComponents;
+                // Where keyword only needs to be appened one time, happends outside of loop
+                StringBuilder whereComponents = new StringBuilder();
```

Comment:
`var whereComponents = new StringBuilder();`

The type is obvious due to the right-hand side, so use `var`.